public/app/services/settingsService.js

```diff
@@ -40,6 +45,200 @@ angular.module('kibibitCodeEditor')
     settings.__defineGetter__('isFullscreen', function() {
       return currentFullscreenState();
     });
+
+    settings.__defineGetter__('editorSettings', function() {
```

Comment:
the `editorSettings` don't need to have a setter and a getter.

it can be exposed as a simple var.